learn/user-guide/security/authentication-and-authorization.md

```diff
@@ -226,26 +333,26 @@ The `http:JwtValidatorConfig` configurations include:
 
 * `issuer` - Expected issuer, which is mapped to `iss`
 * `audience` - Expected audience, which is mapped to `aud`
-* `clockSkewInSeconds` - Clock skew in seconds that can be used to avoid token validation failures due to clock synchronization problems
-* `trustStoreConfig` - JWT trust store configurations
-    * `trustStore` - Trust store used for signature verification
-    * `certificateAlias` - Token-signed public key certificate alias
-* `jwksConfig` - JWKs configurations
-    * `url` - URL of the JWKs endpoint
-    * `cacheConfig` - Configurations related to the cache, which are used to store preloaded JWKs information
-    * `clientConfig` - HTTP client configurations, which call the JWKs endpoint
-* `cacheConfig` - Configurations related to the cache, which are used to store the parsed JWT information
-
-> **Note:** For demonstration purposes, the `ballerinaTruststore.p12` included with the Ballerina runtime is used. In a production deployment, the truststore should only contain the public key certificates of the trusted JWT issuers.
+* `clockSkew` - Clock skew (in seconds) that can be used to avoid token validation failures due to clock synchronization problems
+* `signatureConfig` - JWT signature configurations
+    * `jwksConfig` - JWKs configurations
+        * `url` - URL of the JWKs endpoint
```

Comment:
```suggestion
        * `url` - URL of the JWK endpoint
```

Reply:
This is the technical term we used for that endpoint. So let's keep it as it is.